src/chatClient.cpp

```diff
@@ -2293,8 +2374,13 @@ bool GroupChatRoom::syncWithApi(const mega::MegaTextChat& chat)
     }
     else
     {
-        clearTitle();
-        KR_LOG_DEBUG("Empty title received for group chat %s", Id(mChatid).toString().c_str());
+        // With this contidion some unecesary title changed notification is avoided.
+        // We are notifing all permission changes and only group compositon change must be notified
```

Comment:
_We are still *notifying* title-changes for all *privilege* changes, when only group composition changes must be notified_